common/docker_helper.py

```diff
@@ -0,0 +1,15 @@
+def health_check_info(image_name):
+    import docker
+    from slack_message import slack_message
```

Comment:
You put this file in `common` but `slack_message` is in `dags`. Does this import statement work everywhere you'd use it? Maybe this is related to what you mean by changing the bot's behavior later.

Reply:
I think so, airflow sets the `PYTHONPATH` to include `dags` and `plugins` and I manually add `common`. I plan to gradually move the code that are not specific for the dags to `common`, `slack_message` is one of them, but that change is not directly related to healthcheck, should be put in a different pr

Reply:
That makes sense.